astropy/modeling/tests/example_models.py

```diff
@@ -184,6 +184,15 @@
         'x_values': [1, 10, 100],
         'y_values': [3, 111, 10101],
         'x_lim': [-3, 3]
+     },
+
+    Sersic1D: {
+    'parameters': [1, 20, 4],
+    'x_values': [0.1, 1, 10, 100],
+    'y_values': [2.78629391e+02,   5.69791430e+01,   3.38788244e+00,
+                 2.23941982e-02],
+    'requires_scipy': True,
+    'x_lim': [-100,100]
```

Comment:
Very minor nitpick but these lines should be indented.  I don't particularly care if you update this or not though :)